modules/password/password.go

```diff
@@ -16,12 +16,11 @@ import (
 var matchComplexities = map[string]regexp.Regexp{}
 var matchComplexityOnce sync.Once
 var validChars string
-
 var validComplexities = map[string]string{
 	"lower": "abcdefghijklmnopqrstuvwxyz",
 	"upper": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
 	"digit": "0123456789",
-	"spec":  "_-",
+	"spec":  "[" + `][ !"#$%&'()*+,./:;<=>?@\\^_{|}~` + "`-" + "]",
```

Comment:
You are not using regex in this particular case, so you don't need to escape characters or enclose them in brackets:

```suggestion
	"spec":  `][ !"#$%&'()*+,./:;<=>?@\^_{|}~` + "`-",
```